src/imagery/i.landsat/i.landsat.download/i.landsat.download.html

```diff
@@ -103,29 +95,86 @@ <h2>DESCRIPTION</h2>
 aborting (default is 300 seconds).
 
 <h2>NOTES</h2>
-USGS is still under development and can't be used for searching nor downloading.
+USGS is still under development and can't be used for searching
+nor downloading.
```

Comment:
This sounds weird. It's the usage of the USGS provider which is under development in the eodag end, not USGS per se. Also, I thought only download by id was not possible...